docs/user/features.rst

```diff
@@ -75,6 +75,20 @@ We offer a number of search features:
 
 Learn more about :doc:`/server-side-search/index`.
 
+Traffic Analytics
```

Comment:
I would probably highlight both search & traffic analytics here?

Reply:
I also feel like we're trying to refactor away from this page, so adding more content to it feels wrong. I think it's probably better to draft new pages in our feature list for this, but I guess having them here before refactoring them later is OK?

Reply:
Moved :)